packages/venia-ui/lib/components/Header/accountTrigger.js

```diff
@@ -28,17 +29,20 @@ const AccountTrigger = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
     const rootClassName = accountMenuIsOpen ? classes.root_open : classes.root;
+    const intl = useIntl();
```

Comment:
`const { formatMessage } = useIntl();` would be a lot nicer here

Reply:
and probably in all other instances unless we're consuming other aspects of `useIntl` frequently.